willow/plugins/pillow.py

```diff
@@ -222,6 +223,14 @@ def save_as_heic(self, f, quality=80, lossless=False):
             self.image.save(f, "HEIF", quality=quality)
         return HeicImageFile(f)
 
+    @Image.operation
+    def save_as_avif(self, f, quality=80, lossless=False):
+        if lossless:
+            self.image.save(f, "AVIF", quality=-1, chroma=444)
```

Comment:
nitpick: since we're basing this on pillow-heif, it would be great to document what these magic values mean

https://github.com/bigcat88/pillow_heif/blob/891d3fcef08eb03066f557ccef8fdc2492fd1c40/pillow_heif/options.py#L21-L26

Reply:
also, we need an entry about lossless AVIF at https://github.com/wagtail/Willow/blob/main/docs/guide/save.rst?plain=1#L49

Reply:
Looking at pillow_heif, they admit a small variation of about 1%

https://github.com/bigcat88/pillow_heif/blob/master/tests/write_test.py#L415-L422 and https://github.com/bigcat88/pillow_heif/blob/3798f0df6b12c19dfa8fd76dd6259b329bf88029/tests/helpers.py#L39-L48